core/player.py

```diff
@@ -34,18 +35,17 @@ def add_to_hand_cards(self, cards: List) -> List:
         self.hand_cards.extend(cards)
         return self.hand_cards
 
-    # def add_species(self, hand_card, position: str):
-    #     removed_card = self.hand_cards.pop(hand_card)
-    #     if position == 'left':
-    #         # append left
-    #     if position == 'right':
-    #         self.species.
-
+    # todo wrote: add_species(hand_card, position)
+    # remove the hand card, add a species to the left or right of existing species array.
+    def add_species(self, discard_card: TraitCard, position: str):
+        self.hand_cards.remove(discard_card)
+        if position == 'left':
+            self.species.append(Species())
+        elif position == 'right':
+            self.species.insert(0, Species())
```

Comment:
I would suggest using an Enum for the position, rather than string literals. What do you think?

Reply:
Yes.